__tests__/app.test.js

```diff
@@ -255,4 +255,21 @@ describe("task 9 - /api/comments/:comment_id",()=>{
             })
         })
     })
+})
+describe("task 10 - /api/users",()=>{
+    describe("GET",()=>{
+        test("status code :200 and returns an array of users ",()=>{
+            return request(app).get("/api/users")
+            .expect(200).then(({body})=>{
+                const desiredObj =     {
+                    username: 'butter_bridge',
+                    name: 'jonny',
+                    avatar_url:
+                      'https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg'
+                  }
```

Comment:
A good test though it only checks the first user, it would be a sturdier test if it looped through the array of users and checked the shape of each of the objects.